compose/project.py

```diff
@@ -158,15 +164,19 @@ def get_volumes_from(self, service_dict):
         volumes_from = []
         if 'volumes_from' in service_dict:
             for volume_name in service_dict.get('volumes_from', []):
+                if ':' in volume_name:
+                    service_name, mount_option = volume_name.split(':', 1)
+                else:
+                    service_name, mount_option = volume_name, None
```

Comment:
Could you extract this logic into a function, and do the same with the string-building logic in `Service._get_volumes_from()`? Say, `parse_volumes_from()` and `build_volumes_from()`.